README.md

```diff
@@ -5,6 +5,21 @@
 # Heketi
 Heketi provides a RESTful management interface which can be used to manage the life cycle of GlusterFS volumes.  With Heketi, cloud services like OpenStack Manila, Kubernetes, and OpenShift can dynamically provision GlusterFS volumes with any of the supported durability types.  Heketi will automatically determine the location for bricks across the cluster, making sure to place bricks and its replicas across different failure domains.  Heketi also supports any number of GlusterFS clusters, allowing cloud services to provide network file storage without being limited to a single GlusterFS cluster.
 
+# Important Notice
+
+Due to resource limits on the current project maintainers and general lack of
+contributions we are considering placing Heketi into a near-maintenance mode.
+While we plan to continue to fix issues, correspond with users, review PRs, and
+possibly even make small Quality-of-Life enhancements it is unlikely that any
+major new development will occur. The truth is, this is largely already the
+state of the project and this is our way of letting a wider audience know.
```

Comment:
Do you want to give a date? _state of the project since ..._

Reply:
Thanks for the suggestion but I think I'll skip it... partly at least since I don't really have a concrete date we just kind of realized one day how much the activity had slowed down.